src/__tests__/index.test.ts

```diff
@@ -838,6 +838,13 @@ const vectors: Array<TestVector> = [
       { text: 'EQDZ4T6YyVkg9mb2cAuLEu8iBKbajQAywF', hex: '214d797cc92303dac242b17026e79bbea28eb642f29f0d3582' }
     ]
   },
+  {
+    name: 'NAS',
+    coinType: 2718,
+    passingVectors: [
+      { text: 'n1FF1nz6tarkDVwWQkMnnwFPuPKUaQTdptE', hex: '07f964ff495324635f22c7b486e05d7e67c7af5c' },
```

Comment:
Can you also add `n1sLnoc7j57YfzAVP8tJ3yK5a2i56QrTDdK` to test that the account address also works?

Reply:
Added the test case.

Was adding/removing first 2 bytes while encoding/decoding the address which was [unnecessary](https://github.com/nebulasio/go-nebulas/blob/be27d30e2eab5d1b7dab13e21456f52981acecbf/core/address_test.go#L42), so fixed that.